content/blog/summary-from-the-9th-annual-chapel-implementers-and-users-workshop-chiuw-2022.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Summary from The 9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)
+date: 2022-07-15T19:31:06.645Z
+priority: ""
+author: Michelle Strout and Engin Kayraklioglu
+authorimage: /img/mstrout-headshot-small.jpg
+thumbnailimage: /img/engin.png
+tags:
+  - Chapel Programming Language
+  - GPU Programming
+  - High Performance Computing
+  - Open Source
+  - Arkouda (Data Analytics in Python powered by Chapel)
+---
+**Introduction**
+
+Programming today is complicated by the many kinds of parallelism that exist in everything from phones to laptops to supercomputers.  The open-source Chapel parallel programming language is being developed at HPE with the goals of making programming more productive / easier, while still enabling high performance that takes advantage of the wide variety of parallelism available today.
+
+In this post, we summarize highlights from the recent [9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)](https://chapel-lang.org/CHIUW2022.html).  We are the general chair of the workshop, Michelle Strout, and the program committee chair, Engin Kayraklioglu, and members of the Chapel development team at HPE.  Read on to hear about some exciting applications that are using Chapel productively, the coding day that happened the day before the workshop, updates on the project, and feedback the team received during and after the workshop.
+
+**Applications Written in Chapel**
+
+Programmers generally enjoy trying out new programming languages, but before they lean on a programming language it is always better to see example use cases.  The Chapel programming language is being used productively in a range of application domains: data science, aeronautical simulations, cosmology simulations, and quantum diagonalization to name a few. CHIUW featured the following talks on different Chapel applications:
```

Comment:
Rephrase the first sentence to:

"Programmers generally enjoy trying out new programming languages, but like to see example use cases before they lean on it." 

I also recommend adding in a sentence between this one and the next that says something about how CHIUW gives them a great opportunity to see use cases like this. For instance:

"CHIUW provides them with an opportunity to explore many different use cases."

Reply:
done